variables.md

```diff
@@ -422,6 +422,7 @@ Variable        | Type      | Default value | Details
 XFSTESTS_REPO | string | | repo to install xfstests package
 DEPENDENCY_REPO | string | | ibs/obs repo to install related test package to solve dependency issues. e.g. fio
 XFSTESTS_DEVICE | string | | manually set a test disk for both TEST_DEV and SCRATCH_DEV
+XFSTESTS_SKIP_INSTALL | boolean | 0 | Running xfstests on a disk without xfstests(skip create_hdd). Set this parameter withou PUBLISH_HDD_1 to save disk.
```

Comment:
Typo here: withou -> without. The explanation could be simplified as skipping the install step including xfstests package and dependency package installation.